sdcm/nemesis.py

```diff
@@ -1262,6 +1262,30 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, host_id=None,
         InfoEvent(message="FinishEvent - New Node is up and normal").publish()
         return new_node
 
+    def _add_and_init_new_cluster_node_parallel(self, count, timeout=MAX_TIME_WAIT_FOR_NEW_NODE_UP, rack=0):
+        self.log.info("Adding %s new nodes to cluster...", count)
+        InfoEvent(message=f'StartEvent - Adding {count} new nodes to cluster').publish()
+        new_nodes = self.cluster.add_nodes(
+            count=count, dc_idx=self.target_node.dc_idx, enable_auto_bootstrap=True, rack=rack)
+        self.monitoring_set.reconfigure_scylla_monitoring()
+
+        try:
+            with adaptive_timeout(Operations.NEW_NODE, node=self.cluster.nodes[0], timeout=timeout):
+                self.cluster.wait_for_init(node_list=new_nodes, timeout=timeout, check_node_health=False)
+            self.cluster.set_seeds()
```

Comment:
Messing with the seed isn't really needed anymore, just before starting a node.

Reply:
ok, I'll remove it. See I trimmed original procedure (like setting nemesis target) and I'm not sure of it yet.